### DISCUSSION TITLE
Welcome to pycoral Discussions!

### DISCUSSION THREAD

**Discussion by oberluz** [oberluz/pycoral#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nmaas87** [comment]
No request, but just a huge thanks a lot for your awesome help getting this stuff running again on modern systems 👍🏻
